### DISCUSSION TITLE
Luna crashing

### DISCUSSION THREAD

**Discussion by awjm** [moraroy/NonSteamLaunchers-On-Steam-Deck#127] [Q&A]
Hi all.

Before installing NSL, I did a fresh reset of my Deck.

I installed Edge.

Then went through the NSL process, selecting Epic, Amazon, Uplay, EA, GOG and Luna.

Everything works a dream, aside from Luna. I try to launch and it goes to the 'launching' view (swirling Steam logo). Then it fails/crashes and goes back to the title page.

I've tried different versions of Proton. Resulting in shorter or longer attempts to launch, all with the same result.

Maybe altering the 'launch options' would have an effect?

Thanks.

**Comment by moraroy** [comment]
@awjm I removed edge from the script,Luma requires good chrome to work as well as the other sites. Leave the force compatability empty on the streaming sites. Google Chrome is a Linux program not a proton based program. It will not open or run if you have proton checked.
